docs/source/getting_started/getting_started.md

```diff
@@ -423,7 +390,269 @@ curl -X POST "$URL/users/u1234/personalized_documents" \
+
+## Snippet Id
+
+Snippets are identified by a snippet id consisting of both the document id and a sub id.
+
+Each document is guaranteed to have a snippet with sub id equals `0`. But besides that
+there are no guarantees about the value of the `sub_id`. E.g. a document might have snippet
+id's with the sub ids `0`, `400`, `2334`.
+
+For example if we ingest a document like
+
+```json
+{
+    "id": "12345",
+    "snippet": "some text"
+},
+```
+
+It would have a single snippet with the id `{ "document_id": "12345", "sub_id": 0 }`.
+
+So if we have following semantic search example
+
+```bash
+curl -X POST "$URL/semantic_search" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "document": { "id": "12345" },
+        "include_properties": true
+    }'
+```
+
+we could also use the snippet id instead like shown below
+
+```bash
+curl -X POST "$URL/semantic_search" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "document": { "id": { "document_id": "12345", "sub_id": 0 },
+        "include_properties": true
+    }'
+```
+
+This pattern of replacing document id string with a snippet id struct is also used in all other places
+a snippet id can be used.
+
+Be aware that using `{ "id": "12345" }` represents the whole document but `{ "document_id": "12345", "sub_id": 0 }` represents the first snippet specifically and for some APIs this makes a difference
+for documents which have multiple embeddings as described in the sections below.
+
+## Ingestion
+
+There are two APIs with which you can ingest documents in a way where it can
+produce multiple snippets:
+
+- using the [`split` option with `snippet`](#automatic-document-splitting)
+- using the [`split` option with `file`](#file-upload)
+
+be aware that you need to use sufficient long text for it to be split.
+
+You could use the example from the [`split` option section](#automatic-document-splitting) or use
+your own document, something along ~1000 words will likely generate multiple splits.
+
+## Semantic Search
+
+As mentioned the search results are based on snippets not documents, as such a search
+can yield multiple different snippets of the same document.
+
+The semantic search API can also be used based on a specific snippet (and for systems
+which have documents with multiple embeddings that is recommended):
+
+```bash
+curl -X POST "$URL/semantic_search" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "document": { "id": { "document_id": "effaffe", "sub_id": 2 },
+        "include_properties": true
+    }'
+```
+
+This will return the snippets of which the semantic representation is closest to the
+semantic representation of the second snippet of the given document.
```

Comment:
I don´t like that we say `second snippet`. We said that snippets are not necessarily sequential but we seem to suggest it again (even if 2 would be third).

```suggestion
semantic representation of a snippet of the given document.
```

Reply:
I changed it in a different way. It's still a specific snippet so `a snippet` is a bit to general.